### ISSUE TITLE
Loading Galaxy10 dataset

### ISSUE THREAD

**Issue by ArcaneIrvine** [henrysky/astroNN#19]
"To load images and labels (will download automatically at the first time)"
"# First time downloading location will be ~/.astroNN/datasets/"
**images, labels = load_galaxy10()**

Trying to load the galaxy10 dataset using astroNN but i am getting the following error:
**URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate(_ssl.c:1131)>**

Anyone knows why this is? Thanks in advance.

**Comment by henrysky** [comment]
Can you try again? Maybe just a random certificate error because it is working on my end. Or can you go to https://www.astro.utoronto.ca/~hleung/shared/Galaxy10/Galaxy10_DECals.h5 to see if you can download the h5 file manually (or check if there is any certification warning on your browser). If you can download the .h5 manually, you can download it and move it to the location under `~/.astroNN/datasets/`

**Comment by ArcaneIrvine** [comment]
Yep! Manually works fine, moving it to the location provided will most likely work fine as well. Maybe it has something to do with ubuntu or just my machine.
